assets/prototype/domain/eventEditor/data/queries/currentUser/useCurrentUser.ts

```diff
@@ -0,0 +1,25 @@
+import pathOr from 'ramda/src/pathOr';
+import { useApolloClient } from '@apollo/react-hooks';
+
+import { GET_CURRENT_USER } from './';
+import { ReadQueryOptions } from '../types';
+import { CurrentUserProps } from '../../../../../application/valueObjects/config/types';
+/**
+ * A custom react hook for retrieving CurrentUser
+ */
+const useCurrentUser = (): CurrentUserProps => {
```

Comment:
My understanding is that `Whatever ... Props` is typings for the arguments of a function, not what is returned.
Maybe this should be renamed.

Reply:
I like the naming of `FetchEntitiesResult`  from below.